src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -87,14 +91,15 @@ public class TypicalPersons {
             .withPhone(VALID_PHONE_BOB)
             .withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND).withHouses(HOUSE3).build();
 
-    public static final Person ALI = new BuyerBuilder().withName("Ali York")
+    // Manually added - Buyer's details found in {@code CommandTestUtil}
+    public static final Buyer ALI = new BuyerBuilder().withName("Ali York")
             .withPhone("82937163").withEmail("ali@gmail.com").withHousingType("HDB").withTags("friends").build();
-    public static final Person BEN = new BuyerBuilder().withName(VALID_NAME_BEN).withPhone(VALID_PHONE_BOB)
+    public static final Buyer BEN = new BuyerBuilder().withName(VALID_NAME_BEN).withPhone(VALID_PHONE_BOB)
```

Comment:
Not really an issue, but you can consider following your previous variables names. (eg; `BEN_BUYER`)